### PR TITLE
Add "And many more" to "Built with ReactPHP" linking to our users wiki page

### PR DESCRIPTION
**Pull request by WyriHaximus**
9go8kzc

### CODE REVIEW

**Comment by jsor** [comment]
I think i'd like to have it more like a simple link than a large box.

```diff
diff --git a/src/theme/homepage.html.twig b/src/theme/homepage.html.twig
index e348a69..59d3e81 100644
--- a/src/theme/homepage.html.twig
+++ b/src/theme/homepage.html.twig
@@ -114,6 +114,12 @@ $loop->run();
                 </div>
             {% endfor %}
             </div>
+
+            <p>
+                <a href="https://github.com/reactphp/react/wiki/Users">
+                    And many more on our wiki page &raquo;
+                </a>
+            </p>
         </div>
     </div>
``` 

bildschirmfoto 2017-05-15 um 08 34 11

What do you think?

**Comment by WyriHaximus** [comment]
Will update shortly, @jsor's is great 👍

**Comment by WyriHaximus** [comment]
Updated it to @jsor's suggestion 👍

File: src/theme/homepage.html.twig [hunk 1]
```diff
@@ -113,8 +113,14 @@ $loop->run();
                     </a>
                 </div>
             {% endfor %}
-            </div>
```
Comment (jsor) [inline src/theme/homepage.html.twig]: This `<div>` needs to stay and the new code has to be insied the `<div class="wrapper">` element.

**Comment by WyriHaximus** [comment]
@jsor fixed

**Pull request merged**